recipes/fastjet/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "fastjet" %}
+{% set version = "3.4.0.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fastjet-{{ version }}.tar.gz
+  sha256: 86dea05ce7a030db9fe14e5bf1c0bc46351b6e439f9b9ef6b14a55ceefcc7b7a
+  patches:
+    - 0001-Don-t-overwrite-external-CXXFLAGS-and-LDFLAGS.patch
+    - 0002-Patch-fastjet-contrib-Makefile.in.patch
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script: {{ PYTHON }} -m pip install . -vv
+  missing_dso_whitelist:
+  # https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#whitelisting-shared-libraries
+  # All are under lib/python3.*/site-packages/fastjet/_fastjet_core/lib
+    - $RPATH/libfastjet.so.0
+    - $RPATH/libfastjettools.so.0
+    - $RPATH/libsiscone.so.0
+    - $RPATH/libsiscone_spherical.so.0
+    - $RPATH/libfastjetplugins.so.0
```

Comment:
All of these are in the [`missing_dso_whitelist`](https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#whitelisting-shared-libraries) as they exist under the respective CPython `site-packages` directory but I'm not sure how to include this information in the [`RPATHs` information section](https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#rpaths)

```yaml
build:
  rpaths:
    - lib/
    - ???
```

as the `ORIGIN` is going to be CPython version dependent. Example: `_ext.cpython-311-x86_64-linux-gnu.so` is `$ORIGIN`

```console
$ ls -l build_artifacts/pkg_cache/fastjet-3.4.0.6-py311*/lib/python3.11/site-packages/fastjet
total 10724
-rwxr-xr-x 1 feickert feickert   499440 Feb 20 19:23 _ext.cpython-311-x86_64-linux-gnu.so
drwxr-sr-x 6 feickert feickert     4096 Feb 20 19:24 _fastjet_core
-rw-r--r-- 1 feickert feickert 10059200 Feb 20 19:23 _fastjet.so
-rw-r--r-- 1 feickert feickert    43832 Feb 20 19:23 _generalevent.py
-rw-r--r-- 1 feickert feickert    26065 Feb 20 19:23 __init__.py
-rw-r--r-- 1 feickert feickert    18991 Feb 20 19:23 _multievent.py
drwxr-sr-x 2 feickert feickert     4096 Feb 20 19:24 __pycache__
-rw-r--r-- 1 feickert feickert    20622 Feb 20 19:23 _pyjet.py
-rw-r--r-- 1 feickert feickert    17834 Feb 20 19:23 _singleevent.py
-rw-r--r-- 1 feickert feickert   265128 Feb 20 19:23 _swig.py
-rw-r--r-- 1 feickert feickert     3997 Feb 20 19:23 _utils.py
-rw-r--r-- 1 feickert feickert      204 Feb 20 19:23 version.py
```

and these shared libraries in the whitelist exist under

```
$ ls build_artifacts/pkg_cache/fastjet-3.4.0.6-py311*/lib/python3.11/site-packages/fastjet/_fastjet_core/lib/lib{fastjet,siscone}*.so*
build_artifacts/pkg_cache/fastjet-3.4.0.6-py311hf3e0c80_0/lib/python3.11/site-packages/fastjet/_fastjet_core/lib/libfastjetcontribfragile.so
build_artifacts/pkg_cache/fastjet-3.4.0.6-py311hf3e0c80_0/lib/python3.11/site-packages/fastjet/_fastjet_core/lib/libfastjetplugins.so
build_artifacts/pkg_cache/fastjet-3.4.0.6-py311hf3e0c80_0/lib/python3.11/site-packages/fastjet/_fastjet_core/lib/libfastjetplugins.so.0
build_artifacts/pkg_cache/fastjet-3.4.0.6-py311hf3e0c80_0/lib/python3.11/site-packages/fastjet/_fastjet_core/lib/libfastjetplugins.so.0.0.0
build_artifacts/pkg_cache/fastjet-3.4.0.6-py311hf3e0c80_0/lib/python3.11/site-packages/fastjet/_fastjet_core/lib/libfastjet.so
build_artifacts/pkg_cache/fastjet-3.4.0.6-py311hf3e0c80_0/lib/python3.11/site-packages/fastjet/_fastjet_core/lib/libfastjet.so.0
build_artifacts/pkg_cache/fastjet-3.4.0.6-py311hf3e0c80_0/lib/python3.11/site-packages/fastjet/_fastjet_core/lib/libfastjet.so.0.0.0
build_artifacts/pkg_cache/fastjet-3.4.0.6-py311hf3e0c80_0/lib/python3.11/site-packages/fastjet/_fastjet_core/lib/libfastjettools.so
build_artifacts/pkg_cache/fastjet-3.4.0.6-py311hf3e0c80_0/lib/python3.11/site-packages/fastjet/_fastjet_core/lib/libfastjettools.so.0
build_artifacts/pkg_cache/fastjet-3.4.0.6-py311hf3e0c80_0/lib/python3.11/site-packages/fastjet/_fastjet_core/lib/libfastjettools.so.0.0.0
build_artifacts/pkg_cache/fastjet-3.4.0.6-py311hf3e0c80_0/lib/python3.11/site-packages/fastjet/_fastjet_core/lib/libsiscone.so
build_artifacts/pkg_cache/fastjet-3.4.0.6-py311hf3e0c80_0/lib/python3.11/site-packages/fastjet/_fastjet_core/lib/libsiscone.so.0
build_artifacts/pkg_cache/fastjet-3.4.0.6-py311hf3e0c80_0/lib/python3.11/site-packages/fastjet/_fastjet_core/lib/libsiscone.so.0.0.0
build_artifacts/pkg_cache/fastjet-3.4.0.6-py311hf3e0c80_0/lib/python3.11/site-packages/fastjet/_fastjet_core/lib/libsiscone_spherical.so
build_artifacts/pkg_cache/fastjet-3.4.0.6-py311hf3e0c80_0/lib/python3.11/site-packages/fastjet/_fastjet_core/lib/libsiscone_spherical.so.0
build_artifacts/pkg_cache/fastjet-3.4.0.6-py311hf3e0c80_0/lib/python3.11/site-packages/fastjet/_fastjet_core/lib/libsiscone_spherical.so.0.0.0
```

Reply:
Actually, as `ORIGIN` is `build_artifacts/pkg_cache/fastjet-3.4.0.6-py311*/lib/python3.11/site-packages/fastjet/_ext.cpython-311-x86_64-linux-gnu.so` is there someway to use `$ORIGIN` in the `meta.yml` `rpaths` like

```yaml
build:
...
  rpaths:
    - lib/
    - '$ORIGIN/../fastjet/_fastjet_core/lib/'
```

?

Reply:
Hm. If by trying to remove these we get overlinking errors of the form

```
conda_build.exceptions.OverLinkingError: overlinking check failed 
['  ERROR (fastjet,lib/python3.11/site-packages/fastjet/_fastjet_core/lib/libfastjetcontribfragile.so): lib/python3.10/site-packages/fastjet/_fastjet_core/lib/libfastjettools.so.0 not found in any packages',
...
]
```

how can we stop the conda-forge build scripts from looking across differenet CPython builds that are happening in parallel? It isn't clear to me why this `python3.11` and python `3.10` are having issues here.